pkgs/tools/inputmethods/ibus-engines/ibus-table/default.nix

```
@@ -1,7 +1,7 @@
 { stdenv, fetchFromGitHub
 , autoreconfHook, docbook2x, pkgconfig
 , gtk3, dconf, gobject-introspection
-, ibus, python3 }:
+, ibus, python3, wrapGAppsHook }:
 
 stdenv.mkDerivation rec {
   name = "ibus-table-${version}";
```

While at it, you could also change

```suggestion
  pname = "ibus-table";
```

---

Is this a new(ish) convention? And I guess we still want the line `name = "${pname}-${version}"`?

---

We imported it from `mkPythonModule`. `name = "${pname}-${version}"` is not necessary any more.